### PR TITLE
Fix Missing runtime APIs for Metadata v15

### LINKED ISSUE #2202: Metadata v15 Issues
The upgrade to Metadata v15 is now causing this warning:

`PORTABLEREGISTRY: Unable to determine runtime Call type, cannot inspect sp_runtime::generic::unchecked_extrinsic::UncheckedExtrinsic`

However, there are currently no ill effects of it, nor additional errors. (even with `DEBUG=*`)

Possible Related Issues that might help:
- https://github.com/freeverseio/laos/pull/820
- https://github.com/freeverseio/laos/pull/821
- https://github.com/polkadot-js/apps/issues/10954
- https://github.com/paritytech/substrate-api-sidecar/issues/1516

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is to make sure that Metadata v15 has the correct Runtime API call data

Closes #2202 

# Discussion

Why? I cannot be 100% sure. See the linked issues in #2202 for more details.

From what I can tell, the way rust macros work with context means that this macro is stuck being in the same file as the const that uses it is `pub const VERSION`

# Specifics

- Updated Polkadot js API
- Fixed issue with the runtime APIs and the call in `e2e/capacity/capacity_rpc.test.ts`. Issue noted in code comment
- Moved the runtime apis back into the runtime lib.rs
- Removed the manual definitions of runtimeApis 🎉 

## Testing

- `make start` && `subxt metadata --version 15 -f json | jq .[1].V15.apis` (Before this result would have been empty, now you have the runtime data!)

### CODE REVIEW

File: runtime/frequency/src/lib.rs [hunk 1]
```diff
@@ -1329,4 +1684,9 @@ mod tests {
 			whitelist.contains("26aa394eea5630e07c48ae0c9558cef780d41e5e16056765bc8461851072c9d7")
 		);
 	}
+
+	#[test]
+	fn runtime_apis_are_populated() {
+		assert!(RUNTIME_API_VERSIONS.len() > 0);
```
Comment (enddynayn) [inline runtime/frequency/src/lib.rs:1690]: 👍

**Review by enddynayn — APPROVED** [review]
👍 fabuloso!

File: js/api-augment/types.d.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// Unknown is causing these to be left out of the generated types.
+// Adding them here for now.
```
[the remark this answers is not in the document]
Comment (aramikm) [reply js/api-augment/types.d.ts:2]: Hmm, they implemented `TypeInfo` macro . This is weird!

**Review by aramikm — APPROVED** [review]
I read the code and it looked good afaik.

File: js/api-augment/types.d.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+declare global {
+  type SpConsensusSlotsSlotDuration = bigint;
+  type CommonPrimitivesMsaDelegatorId = bigint;
+  type CommonPrimitivesMsaProviderId = bigint;
```
Comment (JoeCap08055) [inline js/api-augment/types.d.ts:7]: I'm not clear on the purpose of this... but shouldn't these be `u64`, not `bigint`?

Comment (wilwade) [reply js/api-augment/types.d.ts:7]: They are bigint when others on GH have had to do it with `SpConsensusSlotsSlotDuration`, so I just kept with it, but I'll check

File: js/api-augment/definitions/frequency.ts [hunk 4]
```diff
@@ -28,19 +28,8 @@ export default {
       event: 'u8',
       data: 'Vec<u8>',
     },
-  },
-  runtime: {
```
Comment (wilwade) [inline js/api-augment/definitions/frequency.ts:32]: Runtime types are part of the generation now

**Review by JoeCap08055 — APPROVED** [review]

**Review by shannonwells — APPROVED** [review]
I reviewed the code changes.  All looks good.

**Pull request merged**